ui/pages/confirmations/confirmation/templates/create-named-snap-account.js

```diff
@@ -0,0 +1,57 @@
+import {
+  MetaMetricsEventAccountType,
+  MetaMetricsEventCategory,
+  MetaMetricsEventName,
+} from '../../../../../shared/constants/metametrics';
+
+function getValues(pendingApproval, t, actions, _history, _data, contexts) {
```

Comment:
It would be nice to have it written in TS. I am going to check with @matthewwalsh0 to make sure refactoring to TS won't break the system and then I will update it. :)

Reply:
In the meantime, I added the JSDOC [1e1d07f](https://github.com/MetaMask/metamask-extension/pull/25191/commits/1e1d07f470b59ff4c02faf338177c62c5edec0d9) 😄

Reply:
We can resolve this comment, let me know what @matthewwalsh0 says about it but at this point we can leave the TS refactor to a future PR. We do need to be more strict on increase the TS on the codebase and developing code in JS certainly goes into the other direction